src/main/java/org/broadinstitute/hellbender/utils/variant/VariantUtils.java

```diff
@@ -0,0 +1,22 @@
+package org.broadinstitute.hellbender.utils.variant;
+
+/**
+ * VariantUtils contains utility methods for the Variant interface.
+ **/
+public class VariantUtils {
+    static public boolean VariantsAreEqual(Variant v1, Variant v2) {
+        if (v1.getStart() != v2.getStart()) {
+            return false;
+        }
+        if (v1.getEnd() != v2.getEnd()) {
+            return false;
+        }
+        if (!v1.getContig().equals(v2.getContig())) {
```

Comment:
`getContig()` could return null (eg., for unmapped reads), so add a check for that here